app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragment.kt

```diff
@@ -110,4 +233,41 @@ class DownloadFragment : LibraryPageFragment<DownloadItem>(), UserInteractionHan
             )
         }
     }
+
+    private fun getDeleteDownloadItemsOperation(items: Set<DownloadItem>): (suspend () -> Unit) {
+        return {
+            CoroutineScope(IO).launch {
+                downloadStore.dispatch(DownloadFragmentAction.EnterDeletionMode)
```

Comment:
These `EnterDeletionMode` and `ExitDeletionMode` would make more sense to me in `deleteDownloadItems()`

Reply:
Answered during our discussion with Jeff